pkg/credentials/assume_role.go

```diff
@@ -104,6 +104,8 @@ type STSAssumeRoleOptions struct {
 	RoleARN         string
 	RoleSessionName string
 	ExternalID      string
+
+	TokenRevokeType string // Optional, used for token revokation
```

Comment:
```suggestion
	TokenRevokeType string // Optional, used for token revokation (MinIO only extension)
```

Reply:
@harshavardhana Should this comment be on any of the other APIs as well?

Reply:
this should be fine for now.